### DISCUSSION TITLE
GitHub Codespace Machine Specifications Query

### DISCUSSION THREAD

**Discussion by hashfx** [community/community#31331] [Codespaces] [answered]
I have a GitHub Codespace of 8GB RAM and 32GB Storage. Do all my repositories share one machine of above specification or whenever I create a repo, new machine with same specs is allotted each time?

P.S.: In Short, Am I allotted only 1 machine for my Codespace? Do all my repo have 32GB storage capacity individually?

Thanking You

**Comment by greggroth** [comment, marked as answer]
Each time you create a codespace, the machine will have those specifications.  You can see [this documentation](https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/setting-a-minimum-specification-for-codespace-machines) for more information about configuring the requirements of your application.

**Reply by hashfx** [reply]
Thanks for the answer sir @greggroth, my doubts are cleared now!😇
